ereuse_devicehub/resources/device/views.py

```diff
@@ -152,6 +154,33 @@ def query(self, args):
         return query.filter(*args['filter']).order_by(*args['sort'])
 
 
+class DeviceMergeView(View):
+
+    """View for merging two devices
+    Ex. ``device/<id>/merge/id=X``.
```

Comment:
should be ``/devices/<id>/merge/id=X``

Reply:
it's not straightforward to have this view url

Reply:
I mean the base url should be `/devices` (in plural), not `/device` (in singular)

Reply:
it is a typo in the comments

Reply:
ok, just to be sure